Update script.js

LaziukAlena opened the pull request:
В скрипте прописала код, согласно которому: новые элементы добавляться по нажатию Enter; пустые не добавляются; если кликнуть- зачеркивается; если кликнуть повторно  он снова становится обычным; очищать input после добавления нового элемента в список.

LaziukAlena committed:
Update script.js

В скрипте прописала код, согласно которому: новые элементы добавляться по нажатию Enter; пустые не добавляются; если кликнуть- зачеркивается; если кликнуть повторно  он снова становится обычным;
очищать input после добавления нового элемента в список.

mdubik777 commented on script.js:
```diff
@@ -1,3 +1,27 @@
+const inputField = document.querySelector('input');
+const groceriesContainer = document.querySelector('.groceries');
```
Лена,  а почему выборка по классу .groceries? в разметке есть пустой див для элементов списка с id items. Вот в него и нужно добавлять элементы

LaziukAlena committed:
Update script.js

Исправила ошибку. Изменила в выборке класс groceries на пустой div для элементов списка с идентификаторами элементов( #items)
